web-src/src/components/ContentFragmentExportButton.l10n.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+export const intlMessages = {
+  contentFragmentExportDialog: {
+    title: {
+      defaultMessage: 'Export to Content Fragment',
+      id: 'promptResultCard.contentFragmentExportDialogTitle',
+      description: 'Title for Export to Content Fragment dialog',
+    },
+    description: {
+      defaultMessage: 'Export the selected variation as a new content fragment variation.',
+      id: 'promptResultCard.contentFragmentExportDialogDescription',
+      description: 'Description for Export to Content Fragment dialog',
+    },
+    variationNameFieldLabel: {
+      defaultMessage: 'Name',
+      id: 'promptResultCard.contentFragmentExportDialogExportButtonLabel',
+      description: 'Label for the Export button in Export to Content Fragment dialog',
+    },
+    exportButtonLabel: {
+      defaultMessage: 'Export',
+      id: 'promptResultCard.contentFragmentExportDialogExportButtonLabel',
+      description: 'Label for the Export button in Export to Content Fragment dialog',
+    },
+    exportAndOpenButtonLabel: {
+      defaultMessage: 'Export and Open',
+      id: 'promptResultCard.contentFragmentExportDialogExportAndOpenButtonLabel',
+      description: 'Label for the Export and Open button in Export to Content Fragment dialog',
+    },
+    exportButtonProgressAreaLabel: {
+      defaultMessage: 'Exporting...',
+      id: 'promptResultCard.contentFragmentExportDialogExportButtonProgressAreaLabel',
+      description: 'Label for the Export button in Export to Content Fragment dialog when in progress',
+    },
+    cancelButtonLabel: {
+      defaultMessage: 'Cancel',
+      id: 'promptResultCard.contentFragmentExportDialogCancelButtonLabel',
```

Comment:
For consistency, let's use `contentFragmentExportDialog` instead of `promptResultCard`.